### DISCUSSION TITLE
[Help] Change Periodic Scans Time [Docker]

### DISCUSSION THREAD

**Discussion by sastromo** [swingmx/swingmusic#175] [Q&A] [answered]
Hello, (newbie here)
I have swingmusic running on docker and I need to change the time of the periodic scan.. 
How do I do it?

**Comment by cwilvx** [comment, marked as answer]
Hello @sastromo 

Welcome to Swing Music. You can disable them by passing the `-nps` (means `no periodic scans`) or change their interval using

 `-psi <time_in_seconds>` (psi stands for `periodic_scans_interval`.

You can view other options using the `--help` flag.
